index.js

```diff
@@ -14,7 +14,7 @@ const defaults = {
 };
 
 const register = function(server, options) {
-  let tags = _.union(options.tags, defaults.tags);
+  const tags = _.union(options.tags, defaults.tags);
```

Comment:
we can't have a global tags object that is used for all requests, if verbose, it's going to get slow and warning if if its not